metaflow/client/core.py

```diff
@@ -25,13 +26,37 @@
     # python3
     import pickle
 
+# IMPORTANT NOTE ON THE CLIENT
+#
+# The client is not fully consistent particularly when you use the client to
+# query current runs. The previous version of the client accessed data directly
```

Comment:
Before we merge, it would be good to specify which "previous versions".

Reply:
Will do.